tests/test_trainer.py

```diff
@@ -856,6 +857,26 @@ def test_end(self, outputs):
     Trainer().test(model)
 
 
+@mock.patch('argparse.ArgumentParser.parse_args',
+            return_value=argparse.Namespace(**(Trainer.default_attributes)))
```

Comment:
```suggestion
            return_value=argparse.Namespace(**Trainer.default_attributes))
```

Reply:
@Borda this doesn't seem to work, I see a `TypeError: type object argument after ** must be a mapping, not property` while running the unit tests